protected/humhub/modules/admin/views/approval/index.php

```diff
@@ -1,43 +1,121 @@
 <?php
 
-use yii\helpers\Url;
-use yii\helpers\Html;
+use humhub\modules\admin\controllers\ApprovalController;
+use humhub\modules\user\models\ProfileField;
+use humhub\widgets\Button;
+use yii\grid\ActionColumn;
+use humhub\modules\admin\models\UserApprovalSearch;
+use yii\data\ActiveDataProvider;
+use humhub\libs\Html;
 use humhub\widgets\GridView;
 use humhub\modules\user\grid\ImageColumn;
 use humhub\modules\user\grid\DisplayNameColumn;
+use humhub\modules\user\models\User;
+
+/** @var $searchModel UserApprovalSearch */
+/** @var $dataProvider ActiveDataProvider */
+/** @var $availableProfileFields ProfileField[] */
+/** @var $profileFieldsColumns ProfileField[] */
+
+$columns = [
+    [
+        'header' => Html::checkbox('select-all'),
+        'format' => 'raw',
+        'value' => static function (User $model) {
+            return Html::checkbox('ids[]', false, ['id' => 'user-select-'.$model->id, 'value' => $model->id]);
+        }
+    ],
+    ['class' => ImageColumn::class],
+    ['class' => DisplayNameColumn::class],
+    'email',
+];
+foreach ($profileFieldsColumns as $profileField) {
+    $columns[] = [
+        'attribute' => 'profile.'.$profileField->internal_name,
+        'value' => static function (User $model) use ($profileField) {
+            return $profileField->getUserValue($model, false);
```

Comment:
Have you tried what happens when the user enters HTML code into an additional text field? I guess that currently HTML tags are encoded twice.

Reply:
Exactly, it's a mistake, I removed the false param